theia/adapters/usgs/eros_wrapper.py

```diff
@@ -62,40 +62,52 @@ def parse_result_set(cls, result_set):
         return [scene.get('displayId', None) for scene in result_set if 'displayId' in scene]
 
     @classmethod
-    def eros_get(cls, url, request_data, **kwargs):
+    def eros_get(cls, url, request_data, authenticating=False, **kwargs):
         if url != 'login' and (not cls.token()):
             cls.connect()
 
-        new_args = cls.eros_prepare(request_data, **kwargs)
+        new_args = cls.eros_prepare(request_data, authenticating=authenticating, **kwargs)
```

Comment:
non-blocking: this is a little awkward, might change up this var name

Reply:
I have some changes coming up to the eros wrapper. You raise a good point. Will change in those changes!